x-pack/packages/kbn-entities-schema/src/schema/entity.ts

```diff
@@ -11,9 +11,9 @@ import { arrayOfStringsSchema } from './common';
 export const entityBaseSchema = z.object({
   id: z.string(),
   type: z.string(),
-  identity_fields: arrayOfStringsSchema,
+  identity_fields: z.union([arrayOfStringsSchema, z.string()]),
   display_name: z.string(),
-  metrics: z.record(z.string(), z.number()),
+  metrics: z.optional(z.record(z.string(), z.number())),
```

Comment:
perhaps we should remove this?

Reply:
The framework still supports metrics even if the Inventory doesn't use it, let's leave this for now